tests/pkg_fatfs_vfs/Makefile

```diff
@@ -49,6 +48,7 @@ BOARD_WHITELIST := airfy-beacon arduino-due arduino-duemilanove arduino-mega2560
 
 TEST_DEPS += image
 
+include ../Makefile.tests_common
```

Comment:
Also this one before `ifeq ($(BOARD),native)` and even more as no default value is set here.